pkg/clients/tagging/filters_test.go

```diff
@@ -369,3 +440,12 @@ func (apigateway apiGatewayClient) GetRestApisPagesWithContext(_ aws.Context, _
 	fn(apigateway.getRestApisOutput, true)
 	return nil
 }
+
+type apiGatewayV2Client struct {
+	apigatewayv2iface.ApiGatewayV2API
+	getRestApisOutput *apigatewayv2.GetApisOutput
+}
+
+func (apigateway apiGatewayV2Client) GetApisWithContext(_ aws.Context, input *apigatewayv2.GetApisInput, opts ...request.Option) (*apigatewayv2.GetApisOutput, error) {
```

Comment:
@matej-g linter issue remains. It should be fixed here.

Reply:
I committed wrong change 🤦, fixing it now, thanks!